### PR TITLE
Have CI ensure client PRs are formatted.

### PR DESCRIPTION
**Pull request by jmchilton**
Tired of having to ignore and revert unrelated changes when formatting my client changes, this should make sure dev stays clean. People may disagree on whether we should be using prettier - but hopefully we can all agree if we're going to use it we should enforce it for each PR.

### CODE REVIEW

**Comment by nsoranzo** [comment]
Do it make sense to merge it with the `js_lint` job?

**Comment by jmchilton** [comment]
That sounds like an excellent idea.

**Comment by dannon** [comment]
Good idea, thanks!  I'm -0 on the additional Makefile targets, though; I'd rather we encourage usage of the client related actions through the standard package scripts.

**Comment by martenson** [comment]
I see makefile as an educative shortcut that can also teach you how to do it through yarn.

**Comment by jmchilton** [comment]
I use ``make client-watch`` and ``make client-format`` daily currently, obviously I do know how to dip into yarn and cd into client but I prefer not to.

Update: speling fiix

**Comment by dannon** [comment]
@martenson Sure, I guess it cuts both ways -- I think it probably has just as much of an isolation effect keeping people out of the client/package.json guts, but this isn't one I care enough to push for, hence the -0 on that particular bit.  ( to be clear, either way, this is a good and useful change overall)

File: .circleci/config.yml [hunk 1]
```diff
@@ -188,6 +188,13 @@ jobs:
       - run: cd client && yarn install --frozen-lockfile
       - *save_yarn_cache
       - run: cd client && yarn run eslint
+      - run: cd client && yarn run prettier-check
+    steps:
```
Comment (martenson) [inline .circleci/config.yml]: this key is duplicated above

**Comment by dannon** [comment]
The lint will pass once https://github.com/galaxyproject/galaxy/pull/8674 is in.  I PR'd the merge forward since there was a conflict in all the configuration changes recently -- wanted to make sure that got a second pair of eyes.

**Review by nsoranzo — APPROVED** [review]

**Comment by martenson** [comment]
@dannon thanks for merging

**Pull request merged**